src/commands/profile.ts

```diff
@@ -0,0 +1,34 @@
+import { ApplicationCommandOptionType } from 'discord.js';
+
+export const profile: GlobalCommand = {
+	name: 'profile',
+	description: 'Responds with the Profile Picture of the mentioned user',
+	options: [
+		{
+			name: 'user',
+			description: 'The user to get the profile picture of',
+			type: ApplicationCommandOptionType?.User,
+		},
+	],
+	requiresGuild: false,
+	async execute({ interaction, options }) {
+		const user = interaction.user;
+		const param = options[0];
+		if (param?.value === undefined) {
+			await interaction.reply({
+				content: user.avatarURL({ extension: 'png', size: 2048 }) as string,
+				ephemeral: true,
+			});
+		} else if (param?.user?.avatar !== null) {
+			await interaction.reply({
+				content: param?.user?.avatarURL({ extension: 'png', size: 2048 }) as string,
+				ephemeral: false,
+			});
+		} else {
+			await interaction.reply({
+				content: "This user doesn't have an avatar",
+				ephemeral: true,
+			});
+		}
```

Comment:
If we're going to take in the suggestion from Avg on checking that a [URL exists](https://github.com/BYU-CS-Discord/CSBot/pull/33/files#r985131421), this might work better as a function that we can call whenever there is an error getting the avatar at any stage of the process.

Reply:
Was able to get around this by simplifying the code considerably to remove duplicated calls